x-pack/plugins/security/server/routes/authorization/roles/get_all_by_space.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { schema } from '@kbn/config-schema';
+
+import type { RouteDefinitionParams } from '../..';
+import { ALL_SPACES_ID } from '../../../../common/constants';
+import { compareRoles, transformElasticsearchRoleToRole } from '../../../authorization';
+import { wrapIntoCustomErrorResponse } from '../../../errors';
+import { createLicensedRouteHandler } from '../../licensed_route_handler';
+
+export function defineGetAllRolesBySpaceRoutes({
+  router,
+  authz,
+  getFeatures,
+  logger,
+  buildFlavor,
+  config,
+}: RouteDefinitionParams) {
+  router.get(
+    {
+      path: '/internal/security/roles/{spaceId}',
+      options: {
+        tags: ['access:spacesManagement'],
```

Comment:
I think this meant to reference the added `manageSpaces` API action
```suggestion
        tags: ['access:manageSpaces'],
```